docs/basic-features/eslint.md

```diff
@@ -126,7 +126,7 @@ If you're using `eslint-plugin-next` in a project where Next.js isn't installed
 
 ## Linting Custom Directories and Files
 
-By default, Next.js will run ESLint for all files in the `pages/`, `components/`, `lib/`, and `src/` directories. However, you can specify which directories using the `dirs` option in the `eslint` config in `next.config.js` for production builds:
+By default, Next.js will run ESLint for all files in the `pages/`, `app` (only if the experimental `appDir` feature is enabled), `components/`, `lib/`, and `src/` directories. However, you can specify which directories using the `dirs` option in the `eslint` config in `next.config.js` for production builds:
```

Comment:
A list of lint target directories can also be found below. I think this one should be modified as well.

https://github.com/vercel/next.js/blob/d4a3c66479f6ea48c958c663067800d5e3fecf21/docs/api-reference/cli.md?plain=1#L123

Reply:
Good catch @mizdra ! Thanks and done! 😄